content/go/concepts/strings/terms/compare/compare.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: 'Compare()'
+Description: 'Compares two strings in lexicographical order.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+  - 'Characters'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`strings.Compare()`** function is a built-in function in Golang that compares two strings lexicographically. It returns an integer value that indicates the relationship between the two strings.
+
+## Syntax
+
+```pseudo
+strings.Compare(str1, str2)
+```
+
+Where `str1` and `str2` are the two strings that we want to compare.
+- If the two strings are equal, the function returns `0`.
+- If the first string is lexicographically less than the second string, the function returns a `negative` value.
+- If the first string is lexicographically greater than the second string, the function returns a `positive` value.
+

```

Comment:
preface this list

Reply:
@yangc95 I've made the changes, please review. Thank you